schemas/collection/1.0/schema.json

```diff
@@ -12,7 +12,8 @@
     "code",
     "contentEncoding",
     "contentDisposition",
-    "mediaType"
+    "mediaType",
+    "primaryCategory"
```

Comment:
@neha0305verma - we made it required but the test cases are not updated. Please fix the test cases also for this.